### PR TITLE
Hotfix: Fixing a bug in extracting plugins

### PR DESCRIPTION
**Pull request by MSLaguana**
Addressing #106 on top of the previous release directly so we can get a fixed build out.

### CODE REVIEW

**Comment by sdhintGIT** [comment]
how do I get the updated version of remotebuild? getting from npm still has this bug.

**Comment by MSLaguana** [comment]
This pull request hasn't been reviewed yet; once we do that (soon hopefully) we'll publish an update.

File: src/taco-remote/lib/buildManager.ts [hunk 1]
```diff
@@ -328,7 +329,7 @@ class BuildManager {
 
             // Here we want to exclusively extract the contents of the /plugins folder, and we will put it in a separate location
             // Later in taco-remote-lib we will manually merge the plugins into the project to ensure they are added correctly.
-            var localPath: string = path.relative(extractToDir, who.props.path);
+            var localPath: string = path.relative(extractToRemoteDir, who.props.path);
             return !who.props.depth || (who.props.depth === 0 && who.props.Directory) || localPath.split(path.sep)[0] === "plugins";
```
Comment (meenakb) [inline src/taco-remote/lib/buildManager.ts]: Does taco-remote-lib look for the plugins in the right directory and behave as expected?

Comment (MSLaguana) [inline src/taco-remote/lib/buildManager.ts]: Yes, all that side of things works correctly, it was just the initial extraction of files that was broken after a refactoring.

Comment (meenakb) [inline src/taco-remote/lib/buildManager.ts]: :+1:  Looks good to me then.

**Comment by nisheetjain** [comment]
:+1:

**Pull request merged**